src/pages/contact.md

```diff
@@ -7,10 +7,12 @@ description: Contact Camunda
 
 There are various channels where you can reach us.
 
-- We encourage everyone to participate in our **community chat** via [Slack](https://zeebe-io.slack.com/), where you can exchange ideas with other Zeebe and Camunda Cloud users, as well as the product developers.
+- We encourage everyone to participate in our **community** via the [Camunda Cloud community forum](https://forum.camunda.io/) or [Slack](https://camunda-cloud.slack.com/), where you can exchange ideas with other Zeebe and Camunda Cloud users, as well as the product developers.
 
-- If you find a **bug**, please open an issue in this [repository](https://github.com/camunda-cloud/bugs). The bugs have a default template, so please describe what doesn't work in detail.
+- If you find a **bug**, please open an issue in this [repository](https://github.com/camunda-cloud/issues). The bugs have a default template, so please describe what doesn't work in detail.
 
-- For **security-related issues**, please e-mail us at [this address](mailto:cloud-security@camunda.com) so we can solve the problem as quickly as possible.
+- We also accept **feature requests** through a similar process. Please open an issue in this [repository](https://github.com/camunda-cloud/issues). The feature requests have a default template, so please describe what you'd like to see in detail.
 
-- **Feebdack and Support** can be submitted or requested via the corresponding entry in the [Cloud Console navigation menu](../../docs/product-manuals/cloud-console/troubleshooting/feedback-and-support). If you have a **license agreement** with us you will be redirected to the [Support queue](https://jira.camunda.com/projects/SUPPORT/) at Camunda. In the Free Plan, contact can be made using the internal form:
+- For **security-related issues**, please e-mail us at [this address](mailto:cloud-security@camunda.com) so we can solve the problem as quickly as possible. Do not use GitHub for security-related issues.
```

Comment:
@felix-mueller @menski please update the Support and Security process, links, and contact info.

Reply:
These are being worked in a Google doc, I'll take care of this as we solidify the processes internally.